src/storage/in_memory.rs

```diff
@@ -10,36 +10,62 @@ type Accounts = HashMap<Address, EvmAccount, BuildAddressHasher>;
 
 /// A storage that stores chain data in memory.
 #[derive(Debug, Default, Clone)]
-pub struct InMemoryStorage {
+pub struct InMemoryStorage<'a> {
     accounts: Accounts,
+    bytecodes_ref: Option<&'a AHashMap<B256, EvmCode>>,
```

Comment:
Just remove the previous `bytecodes` and make this one `bytecodes`? For empty mock bytecodes just pass `None` in.

Reply:
that makes perfect sense